From 6942fce93037babae3b6e96d549b4b9ce9e73f4a Mon Sep 17 00:00:00 2001
From: Christoph Reiter <reiter.christoph@gmail.com>
Date: Thu, 15 Dec 2022 20:30:32 +0100
Subject: [PATCH 37/N] Revert "Cygwin: x86_64: add wmemset assembler entry
 point"

This partly reverts commit 188d5f6c9ad5cbbd6f0fcb9aaf15bc9870597910.

See https://cygwin.com/pipermail/cygwin/2022-December/252699.html
---
 winsup/cygwin/x86_64/memset.s | 8 --------
 1 file changed, 8 deletions(-)

diff --git a/winsup/cygwin/x86_64/memset.s b/winsup/cygwin/x86_64/memset.s
index d3b95ec..ac73b6a 100644
--- a/winsup/cygwin/x86_64/memset.s
+++ b/winsup/cygwin/x86_64/memset.s
@@ -67,11 +67,3 @@ L1:     rep
 	movq	16(%rsp),%rdi
 	ret
 	.seh_endproc
-
-	.globl  wmemset
-	.seh_proc wmemset
-wmemset:
-	.seh_endprologue
-	shlq	$1,%r8		/* cnt * sizeof (wchar_t) */
-	jmp	memset
-	.seh_endproc
